### PR TITLE
Ria 3731 remove representation

### PR DESCRIPTION
**Pull request by bradsorour**
### JIRA link (if applicable) ###

[RIA-3731](https://tools.hmcts.net/jira/browse/RIA-3731)

### Change description ###

See Jira

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/sms/AppellantRemoveRepresentationPersonalisationSms.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            .orElseThrow(() -> new IllegalStateException("appellantMobileNumber is not present")));
+    }
+
+    @Override
+    public String getReferenceId(Long caseId) {
+        return caseId + "_REMOVE_REPRESENTATION_APPELLANT_SMS";
+    }
+
+    @Override
+    public Map<String, String> getPersonalisation(AsylumCase asylumCase) {
```
Comment (makowalski) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/sms/AppellantRemoveRepresentationPersonalisationSms.java:52]: you can use different method with callback object
```
Map<String, String> getPersonalisation(Callback<AsylumCase> callback)
```
so you don't need to add additional field `CCD_CASE_ID`

Comment (bradsorour) [reply src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/sms/AppellantRemoveRepresentationPersonalisationSms.java]: Now using the above method and I've also moved the unnecessary CCD_CASE_ID field.

**Review by makowalski — APPROVED** [review]
LGTM

**Review by Lee365 — DISMISSED** [review]
LGTM

**Review by Srihari-Nakkala — DISMISSED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**